masterlist.yaml

```diff
@@ -7724,6 +7724,12 @@ plugins:
         subs: [ 'Falskaar' ]
         condition: 'active("Falskaar.esm") and not active("You Hunger - Falskaar.esp")'
 
```

Comment:
Plugins from the same mod don't really need a space between them, unless there are a lot of patches.

Reply:
Sorry. I don't get it. Do you mean the bash tags are unnecessary for the patches? Could you tell me what you meant by 'a space between them'?

Reply:
> Could you tell me what you meant by 'a space between them'?

The newline between the two entries.

![image](https://user-images.githubusercontent.com/1944639/64352065-48fc2f00-cff3-11e9-878b-8520a5cf27c2.png)

Reply:
Okay. I removed the newline.